recipes-security/optee/optee-os-tegra/0001-allow-setting-sysroot-for-libgcc-lookup.patch

```diff
@@ -0,0 +1,36 @@
+From 03eed2563105b3e8e452f7326024967d9d27bd83 Mon Sep 17 00:00:00 2001
+From: Ilies CHERGUI <ilies.chergui@gmail.com>
+Date: Fri, 26 Aug 2022 01:27:30 +0100
+Subject: [PATCH 1/2] allow setting sysroot for libgcc lookup
+
+This patch is based on the following commit:
```

Comment:
This patch might not be needed based on https://lists.yoctoproject.org/g/meta-arm/message/3668 (since it is also OP-TEE 3.16.0 based).

Reply:
Super, Thanks a lot for sharing this link.
I certainly miss this. 
I will check that then remove `LIBGCC_LOCATE_CFLAGS`